codespell_lib/data/dictionary.txt

```diff
@@ -18065,6 +18070,7 @@ isconnection->isconnected
 iscrated->iscreated
 iself->itself
 iselfe->itself
+isenough->is enough
```

Comment:
Little worried this will bring up unnecessary false positives  
https://grep.app/search?q=%5Cbisenough%5Cb&regexp=true

Reply:
Well, it's gone now. I hope there isn't anything else to not approve this PR.

Reply:
Please re-add this but put it in the code dictionary if you've seen it in the wild, it won't affect documentation in the same way (and can always be skipped there if required).